internal/publish/publish.go

```diff
@@ -329,7 +284,7 @@ func (p *defaultPublisher) publishWithClient(
 		return err
 	}
 
-	err = p.checkConfiguration(client)
+	err = p.preFlightChecks(client)
```

Comment:
As discussed, this was renamed to reduce confusion and increase maintainability.